L/libdeflate/bundled/patches/fix_makefile.patch

```diff
@@ -0,0 +1,27 @@
+From 2fa25d0f8eee2a467d17dc92f5b25285a8582c74 Mon Sep 17 00:00:00 2001
```

Comment:
I encourage you submitting the patch upstream :slightly_smiling_face:

Reply:
Will do. What is the correct way to change the build script (adding the upsteam information about the patch as a comment) without recomputing the binaries?

Reply:
If you add `[skip ci]` in the commit message, the ci job won't run on master